CGPs/cgp-0143/mainnet.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+      "contract": "GoldToken",
+      "address": "0x471EcE3750Da237f93B8E339c536989b8978a438",
+      "function": "increaseAllowance",
```

Comment:
Just wanted to verify that `increaseAllowance` is the correct method to be called? Im guessing it is but other requests for funds usually use `approve`

```
[
    {
    "contract": "GoldToken",
    "function": "approve",
    "args": ["0xda2069f47D252121c2288301D6EF50B87220A693","1600000000000000000000000"],
    "value": "0"
    }
]
```